src/libs/sfdk/dockercontainer_p.h

```diff
@@ -0,0 +1,83 @@
+/****************************************************************************
+**
+** Copyright (C) 2019 Open Mobile Platform LLC.
+** Contact: http://jolla.com/
+**
+** This file is part of Qt Creator.
+**
+** GNU Lesser General Public License Usage
+** Alternatively, this file may be used under the terms of the GNU Lesser
+** General Public License version 2.1 as published by the Free Software
+** Foundation and appearing in the file LICENSE.LGPL included in the
+** packaging of this file.  Please review the following information to
+** ensure the GNU Lesser General Public License version 2.1 requirements
+** will be met: http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html.
+**
+** Other Usage
+**
+** Alternatively, this file may be used in accordance with the terms and
+** conditions contained in a signed written agreement between you and
+** The Qt Company.
+**
+****************************************************************************/
+#ifndef DOCKERCONTAINER_P_H
+#define DOCKERCONTAINER_P_H
+
+#include "virtualmachine_p.h"
+#include "dockercontainer.h"
+
+namespace Sfdk {
+
+class DockerContainerPrivate : public VirtualMachinePrivate
+{
+    Q_DECLARE_PUBLIC(DockerContainer)
+
+public:
+    DockerContainerPrivate(VirtualMachine* virtualMachine);
```

Comment:
Initialization should be done in public class' ctor instead. Private ctor can be inherited then:

```
using VirtualMachinePrivate::VirtualMachinePrivate;
```

Reply:
fixed